force-app/main/default/aura/HH_AddressMgr/HH_AddressMgrController.js

```diff
@@ -33,6 +33,13 @@
         component.set('v.isExistingAddrSectionOpen', true);
     },
 
+    handleChangeAddressToggle : function(component) {
+        if (component.get('v.useDefaultFocus')) {
```

Comment:
using a boolean attribute here to ensure that the programmatic focus is only set on the modal once and does not steal focus from the accordion interactive elements when they are clicked to expand the accordion sections